libglusterfs/src/glusterfs/mem-pool.h

```diff
@@ -21,6 +21,12 @@
 #include <string.h>
 #include <stdarg.h>
 
+#include <sys/param.h> /* roundup */
+#ifndef roundup
+/* not a standard, so some platforms may not have it. Define a generic one */
+#define roundup(x, y) ((((x) + ((y)-1)) / (y)) * (y))
```

Comment:
I think you should convert roundup(x, y) as xavi suggested
#define roundup(_x, _y) (((_x) + (_y) - 1) & ~((_y) - 1))

Reply:
Will do.

Reply:
Pushed this.